docs/custom-domain/README.md

```diff
@@ -1,105 +1,128 @@
-# Using Custom Domains with Azure Container Registry
+# Use a custom domain with Azure Container Registry (private preview)
 
-Every ACR is accessed using its login server. If you have a registry called `myregistry`, you access it using its default hostname, `myregistry.azurecr.io` (in Azure Public Cloud.) As a customer belonging to an organization, you may prefer to access your registry using a custom domain that is associated with your organization, for instance, `container-registry.contoso.com`.
+This article describes the steps to set up the endpoints of an Azure container registry using your own custom domain name, such as **contoso.com**.
 
-The following steps describe how you can achieve this.
+By default, every Azure container registry is accessed externally using its login server name. If you have a registry called `myregistry`, you access it using its default hostname, `myregistry.azurecr.io` (in the Azure Cloud). As a customer belonging to an organization, you may prefer to access your registry using a custom domain name that is associated with your organization, for example, `myregistry.contoso.com`.
+
+**Important - Using a custom domain name in Azure Container Registry is currently a private preview feature**. It can only be enabled by preparing certificates, a Premium container registry, and an Azure key vault as described in this article, and then opening an Azure support request to complete the configuration. 
+
+## Limitations
+
+* This capability is provided without a service level agreement, and isn't currently recommended for production workloads.
```

Comment:
@sajayantony
Are we not willing to support customers with custom domains?
Or, are we only willing to support a subset of customers, so not all requests will be answered? Just seems the only customers that would want custom domains would be customers in production environments that need the extra security.